pallets/allocations/src/lib.rs

```diff
@@ -67,17 +75,33 @@ pub mod pallet {
 		#[pallet::constant]
 		type ExistentialDeposit: Get<BalanceOf<Self>>;
 
+		type OracleMembersFilter: Contains<Self::AccountId>;
```

Comment:
Trivial suggestion, feel free to skip: rename OracleMembersFilter to OracleMembers or Oracles.

Reply:
Done